bitnami/harbor/templates/registry/registry-dpl.yaml

```diff
@@ -61,41 +71,71 @@ spec:
-            timeoutSeconds: {{ .Values.registry.registry.livenessProbe.timeoutSeconds }}
-            successThreshold: {{ .Values.registry.registry.livenessProbe.successThreshold }}
-            failureThreshold: {{ .Values.registry.registry.livenessProbe.failureThreshold }}
+            initialDelaySeconds: {{ .Values.registry.server.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.registry.server.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.registry.server.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.registry.server.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.registry.server.livenessProbe.failureThreshold }}
+          {{- else if .Values.registry.server.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.registry.server.customLivenessProbe "context" $) | nindent 12 }}
           {{- end }}
-          {{- if .Values.registry.registry.readinessProbe.enabled }}
+          {{- if .Values.registry.server.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
               path: /
               port: registry
-            initialDelaySeconds: {{ .Values.registry.registry.readinessProbe.initialDelaySeconds }}
-            periodSeconds: {{ .Values.registry.registry.readinessProbe.periodSeconds }}
-            timeoutSeconds: {{ .Values.registry.registry.readinessProbe.timeoutSeconds }}
-            successThreshold: {{ .Values.registry.registry.readinessProbe.successThreshold }}
-            failureThreshold: {{ .Values.registry.registry.readinessProbe.failureThreshold }}
+            initialDelaySeconds: {{ .Values.registry.server.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.registry.server.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.registry.server.readinessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.registry.server.readinessProbe.successThreshold }}
+            failureThreshold: {{ .Values.registry.server.readinessProbe.failureThreshold }}
+          {{- else if .Values.registry.server.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.registry.server.customReadinessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.registry.server.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.registry.server.lifecycleHooks "context" $) | nindent 12 }}
           {{- end }}
-          {{- if .Values.registry.registry.resources }}
-          resources: {{- toYaml .Values.registry.registry.resources | nindent 12 }}
+          {{- if .Values.registry.server.resources }}
+          resources: {{- toYaml .Values.registry.server.resources | nindent 12 }}
           {{- end }}
           env:
             - name: BITNAMI_DEBUG
```

Comment:
Could we review DEBUG feature ? I saw sometime we use `BITNAMI_DEBUG` and `.Values.<image>.debug` and other times we use something like `.Values.loglevel`

Reply:
I don't have an strong opinion on this. I would leave it in the image because it is a Bitnami value and not an application value itself

Reply:
Agree.